app/controllers/followers_controller.rb

```diff
@@ -1,4 +1,25 @@
 class FollowersController < ApplicationController
   def create
+    @event = find_event
+    user = current_user
+    follower = Follower.new
+    #follower = @event.followers.new
+    follower.event_id = @event.id
+    follower.user_id = user.id
+    if !(Follower.exists?(event_id: follower.event_id, user_id: follower.user_id))
+      follower.save
+      flash[:dange] = "Event followed"
```

Comment:
nope :D tak też działa. Btw. do tych commitów już Adam robił code review, zacznij od "Add number of participants to Event".